src/utils/getPreperedPeople.ts

```diff
@@ -0,0 +1,19 @@
+import { Person } from '../types';
+
+const getPersonByName = (
+  people: Person[],
+  searchingPersonName: string,
+) => people.find(person => person.name === searchingPersonName);
+
+export const getPreperedPeople
```

Comment:
Where do we use this function ?

Reply:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/83500664/216621717-a4cd8ba0-b4d1-43c8-a61d-8be4ec9982ff.png">
here